desc/geometry/surface.py

```diff
@@ -60,7 +60,6 @@ def __init__(
         NFP=1,
         sym="auto",
         rho=1,
-        grid=None,
```

Comment:
these were unused arguments, so I got rid of them.

Reply:
thanks, forgot to remove them in #583